public/app/dashboard/lookingForCollaborators.html

```diff
@@ -0,0 +1,17 @@
+<div class="profile-looking-for-collaborators-wrapper">
+  <div data-ng-if="isMyProfile">
```

Comment:
@johncrossman It's a good question that might apply to the profile page more broadly; I think non-member admin users can also edit their personal description, etc?